packages/cli/generators/example/index.js

```diff
@@ -30,6 +30,13 @@ const EXAMPLES = {
     'controllers, interceptors, and observers.',
   'lb3-application':
     'An example LoopBack 3 application mounted in a LoopBack 4 project.',
+  'file-upload': 'An example showing how to expose APIs to upload files.',
+  'access-control-migration':
+    'An access control example migrated from the LoopBack 3 repository ' +
+    'loopback-example-access-control.',
+  'metrics-prometheus':
+    'An example illustrating metrics provided by ' +
```

Comment:
Do we want to keep it shorter, e.g.
`An example illustrating metrics using Prometheus`? 

Once the user downloaded the example, they will see the README that which LB4 module it's enabling this.